src/plugins/telemetry/include/plugins/telemetry/telemetry.h

```diff
@@ -855,13 +863,27 @@ class Telemetry : public PluginBase {
      */
     typedef std::function<void(RCStatus rc_status)> rc_status_callback_t;
 
+    /**
+     * @brief Callback type for UTM global position updates.
```

Comment:
Since this only contains the time, I would rename the method to what it is, e.g. `unix_epoch_time`.